### ISSUE TITLE
False negative for unpacked dictionary argument

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#5309]
The following code should produce a type error but does not.

```python
def func1(**kwargs: int): ...
func1(**{"": ""})
```

**Comment by erictraut** [comment]
This will be addressed in the next release.

**Comment by erictraut** [comment]
This is included in pyright 1.1.315, which I just published. It will also be included in this week's insiders release of pylance.
